### PR TITLE
add README for docker usage

### PR DESCRIPTION
**Pull request by egze**
Preview README md — cli 2020-10-26 15-06-25

### CODE REVIEW

File: openapi-generator/templates/cli/README.handlebars [hunk 1]
```diff
@@ -53,7 +53,19 @@ Use the `pull` command to download the most recent locale files back into your p
...
+
+You can also use `phrase` through a docker image, without installing the cli on your computer.
+
+    $ docker run --rm phrase/phrase-cli:2.0.15 help
+
+Some commands are interactive and require the `-it` flag.
+
+    $ docker run -it phrase/phrase-cli:2.0.15 init
+
+You can also use the docker image as base for you more complex images.
```
Comment (tobstarr) [inline openapi-generator/templates/cli/README.handlebars:66]: either strip "you" (I would do this) or change it to "your"

File: openapi-generator/templates/cli/README.mustache [hunk 2]
```diff
@@ -47,7 +47,19 @@ Use the `pull` command to download the most recent locale files back into your p
...
+
+You can also use `phrase` through a docker image, without installing the cli on your computer.
+
+    $ docker run --rm phrase/phrase-cli:2.0.15 help
+
+Some commands are interactive and require the `-it` flag.
+
+    $ docker run -it phrase/phrase-cli:2.0.15 init
+
+You can also use the docker image as base for you more complex images.
```
Comment (tobstarr) [inline openapi-generator/templates/cli/README.mustache:60]: please remove "you" (see above)

**Review by tobstarr — CHANGES_REQUESTED** [review]
grammar

File: openapi-generator/templates/cli/README.handlebars (hunk 1 above)
Comment (egze) [reply openapi-generator/templates/cli/README.handlebars]: done

File: openapi-generator/templates/cli/README.mustache (hunk 2 above)
Comment (egze) [reply openapi-generator/templates/cli/README.mustache]: done

**Review by tobstarr — APPROVED** [review]

**Pull request merged**